### ISSUE TITLE
Add privacy policy route

### ISSUE THREAD

**Issue by ijdickinson** [epimorphics/hmlr-linked-data#24]
I'm not 100% sure of this, but I believe that the `/accessibility` route is handled by the front-end web server config, to map to `/app/landing/doc/accessibility`. Assuming this is correct, can we please also add a similar route for `/privacy` to map to `/app/landing/doc/privacy`?

**Comment by der** [comment]
Yes, can see a rewrite statement for `/accessibility` so should be easy to replicate for `/privacy`

**Comment by der** [comment]
@ijdickinson The current rewrite is `/accessibility` to `/app/root/doc/accessibility` rather than `/app/landing/doc/accessibility`

Should the `/privacy` one match that?

**Comment by ijdickinson** [comment]
> Should the /privacy one match that?

Yes please. I should have copy/pasted, rather than typed it from memory!

**Comment by der** [comment]
Cookbook changed and applied to preprod. If that looks right (not sure how to test) then can apply to production.
Left it set up so the next configure on production will include this change.

**Comment by ijdickinson** [comment]
We updated preprod yesterday, and the `/privacy` route looks good.
